lib/fog/azurerm/requests/compute/delete_managed_disk.rb

```diff
@@ -4,11 +4,15 @@ module Compute
     class AzureRM
       # Real class for Compute Request
       class Real
-        def delete_managed_disk(resource_group_name, disk_name)
+        def delete_managed_disk(resource_group_name, disk_name, async)
```

Comment:
Check whether the call sent is an asynchronous call or not and return the response accordingly. Check the delete method for server [here](https://github.com/fog/fog-azure-rm/blob/master/lib/fog/azurerm/requests/compute/delete_virtual_machine.rb#L6).